software-modules/internet/chromium/chromium.desktop

```diff
@@ -3,7 +3,7 @@ Version=1.0
 Name=Chromium Web Browser
 GenericName=Web Browser
 Comment=Access the Internet
-Exec=/usr/bin/chromium %U
+Exec=/usr/lib/chromium/chromium --show-component-extension-options --enable-gpu-rasterization --no-default-browser-check --disable-pings --media-router=0 --enable-remote-extensions --flag-switches-begin --flag-switches-end
```

Comment:
Are you sure we want to Enable the GPU rasterization? 
It was exactly what used to trigger the Nvidia driver issue. Probably rendering with CPU or disabling the GPU acceleration would be better for compat?

Reply:
Not really sure, those were the default arguments chromium took, I just copy and pased them, will remove the argument.